common/Mesh-Network-Joining-Process-Comparison.md

```diff
@@ -0,0 +1,540 @@
+
+The Unprovisioned Device beacon is used by devices that are unprovisioned to allow them to be discovered by a Provisioner.   
+
+<div align="center">
+  <img src="files/CM-Mesh-Network-Comparison/joining-process-unprovisioned-device-beacon-format.png">  
+</div>  
+<div align="center">
+  <b>  Unprovisioned device beacon format </b>
+</div>  
+<br>
+
+The out-of-band (OOB) Information field in the Unprovisioned Device beacon is used to help drive the provisioning process by indicating the availability of OOB data. A separate non-connectable advertising packet with a ***Uniform Resource Identifer*** (URI) data type that points to OOB information will be advertised if use OOB. In this case, to allow the association of the advertised URI with the Unprovisioned Device beacon, the beacon may contain an optional 4-octet URI Hash field.   
+
+Below is a example of the Unprovisioned Device beacon of the device that supports PB-ADV, the Device UUID is ```{0x53,0x69,0x6C,0x61,0x62,0x73,0x44,0x65,0x76,0x2D,0x95,0x8D,0x64,0x57,0x0B,0x00}``` and it doesn't use the OOB, so the URI Hash field is not included there.   
+
+<div align="center">
+  <img src="files/CM-Mesh-Network-Comparison/joining-process-unprovisioned-device-beacon-data.png">  
+</div>  
+<div align="center">
+  <b>  Unprovisioned device beacon data </b>
+</div>  
+<br>
+
+#### 3.4.1.2. PB-GATT
+PB-GATT is a provisioning bearer used to provision a device using Proxy PDUs to encapsulate Provisioning PDUs within the Mesh Provisioning Service. And the PB-GATT is provided for support when a Provisioner does not support PB-ADV due to limitations of the application interfaces.   
+
+### 3.4.2. Invitation 
+In this step, the Provisioner shall sends an invitation to the device to be provisioned, in the form of a *Provisioning Invite PDU*. And the Beaconing device responds with information about itself in a provisioning Capabilities PDU.   
+The figure below illustrates the message sequence for provisioning invitation.   
+
+<div align="center">
+  <img src="files/CM-Mesh-Network-Comparison/joining-process-provisioning-invitation.png">  
+</div>  
+<div align="center">
+  <b>  Unprovisioned device beacon data </b>
+</div>  
+<br>
+
+A Provisioner sends the *Provisioning Invite PDU* to indicate to the device that the provisioning process is starting. The format of the parameters for this PDU is defined as below.   
+
+<div align="center">
+  <img src="files/CM-Mesh-Network-Comparison/joining-process-btmesh-provisioning-invite-pdu.png">  
+</div>  
+<div align="center">
+  <b>  Provisioning Invite PDU parameters format </b>
+</div>  
+<br>
+
+The *Attention Timer state* is generally intended to allow an element to attract human attention during provisioning, e.g., LED blinks. A device may not support the Attention Timer, and the Attention Timer state shall always be set to zero.   
```

Comment:
A device may not support the Attention Timer, **then** the Attention Timer state shall always be set to zero.